core/src/banking_stage/consumer.rs

```diff
@@ -205,27 +209,16 @@ impl Consumer {
     }
 
     fn process_packets_transactions(
+        &self,
         bank: &Arc<Bank>,
         bank_creation_time: &Instant,
-        committer: &Committer,
-        transaction_recorder: &TransactionRecorder,
         sanitized_transactions: &[SanitizedTransaction],
         banking_stage_stats: &BankingStageStats,
-        qos_service: &QosService,
         slot_metrics_tracker: &mut LeaderSlotMetricsTracker,
-        log_messages_bytes_limit: Option<usize>,
     ) -> ProcessTransactionsSummary {
-        // Process transactions
-        let (mut process_transactions_summary, process_transactions_us) =
-            measure_us!(Self::process_transactions(
-                bank,
-                bank_creation_time,
-                sanitized_transactions,
-                committer,
-                transaction_recorder,
-                qos_service,
-                log_messages_bytes_limit,
-            ));
+        let (mut process_transactions_summary, process_transactions_us) = measure_us!(
+            self.process_transactions(bank, bank_creation_time, sanitized_transactions,)
```

Comment:
nit: extra comma at: `sanitized_transactions,)` (naughty me knows this is rustfmt's bug...)

Reply:
done: 7710e7f5d0